ang/crmUi.js

```diff
@@ -642,7 +642,7 @@
             $timeout(function () {
               // ex: msg_template_id adds new item then selects it; use $timeout to ensure that
               // new item is added before selection is made
-              element.select2('val', ngModel.$viewValue);
+              element.select2('val', ngModel.$modelValue);
```

Comment:
I tried locally changing crmUiSelect to $modelValue, and the existing single-select fields still worked. Also, there's an instance of crmEntityRef (a single-select field for campaign_id) which still seems to work with this patch.

A few lines below, there's a second reference to $viewValue. Any thoughts on whether that should be changed too?